sympy/solvers/ode.py

```diff
@@ -8675,4 +8709,4 @@ def _nonlinear_3eq_order1_type5(x, y, t, eq):
     sol1 = dsolve(diff(u(t),t) - (u*(c*F2-b*F3)).subs(v,y_x).subs(w,z_x).subs(u,u(t))).rhs
     sol2 = dsolve(diff(v(t),t) - (v*(a*F3-c*F1)).subs(u,x_y).subs(w,z_y).subs(v,v(t))).rhs
     sol3 = dsolve(diff(w(t),t) - (w*(b*F1-a*F2)).subs(u,x_z).subs(v,y_z).subs(w,w(t))).rhs
-    return [sol1, sol2, sol3]
+    return [sol1, sol2, sol3]
```

Comment:
I'm not sure what you've changed here. It looks like you've added or deleted a newline. Can you change it back to how it was?

Reply:
I don't know why it's showing this when there is only one instance of it.